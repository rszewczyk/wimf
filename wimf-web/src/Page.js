// @flow
import React from "react";
import { css } from "glamor";

type PageProps = {
  children: any
};

const pageStyle = css({
  display: "flex",
  maxHeight: "100vh",
  overflow: "hidden"
});

export default function Page(props: PageProps) {
  return (
    <div {...pageStyle}>
      {props.children}
    </div>
  );
}
