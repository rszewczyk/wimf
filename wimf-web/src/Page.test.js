import React from "react";
import renderer from "react-test-renderer";
import Page from "./Page";

it("renders correctly", () => {
  expect(
    renderer
      .create(
        <Page>
          <h1>hello!</h1>
          <p>foo</p>
        </Page>
      )
      .toJSON()
  ).toMatchSnapshot();
});
