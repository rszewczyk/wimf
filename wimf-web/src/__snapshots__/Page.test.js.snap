// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  data-css-1upbc3x=""
>
  <h1>
    hello!
  </h1>
  <p>
    foo
  </p>
</div>
`;
